src/BinOp.chpl

```diff
@@ -1044,4 +1045,585 @@ module BinOp
     omLogger.error(getModuleName(),getRoutineName(),getLineNumber(),errorMsg);
     return new MsgTuple(errorMsg, MsgType.ERROR);
   }
+
+  proc doBigIntBinOpvv(l, r, op: string) throws {
+    var max_bits = max(l.max_bits, r.max_bits);
+    var max_size = 1:bigint;
+    var has_max_bits = max_bits != -1;
+    if has_max_bits {
+      max_size <<= max_bits;
+    }
+    ref la = l.a;
+    ref ra = r.a;
+    var tmp = makeDistArray(la.size, bigint);
+    // i think tmp should be initialized to 0 but just to be sure
+    tmp = 0:bigint;
```

Comment:
I ran the test code below (pulling out of `makeDistDom` and `makeDistArray`). `tmp` will indeed initialize to an array of 0s with size `la.size`.

```chapel
module Test_Mod{
    use BigInteger;
    public use BlockDist;
    
    proc main() throws {
        var size = 5;
        var D = {0..#size} dmapped Block(boundingBox={0..#size});
        var a: [D] bigint;
        writeln(a);
    }
}
```

The output from the above code: `0 0 0 0 0`. I would recommend removing `tmp = 0:bigint`.